src/kemal/init_handler.cr

```diff
@@ -9,6 +9,7 @@ module Kemal
     def call(context : HTTP::Server::Context)
       context.response.headers.add "X-Powered-By", "Kemal" if Kemal.config.powered_by_header?
       context.response.content_type = "text/html" unless context.response.headers.has_key?("Content-Type")
+      context.response.headers.add "Date", Time.local.to_rfc2822
```

Comment:
**suggestion:** `Time#to_rfc2822` produces a date format in UTC. So there's no local time information involved.
```suggestion
      context.response.headers.add "Date", Time.utc.to_rfc2822
```
`Time.utc` is sufficient. And it's typically more performant than `Time.local`.